modyn/trainer_server/internal/trainer/remote_downsamplers/abstract_remote_downsample_strategy.py

```diff
@@ -62,3 +66,7 @@ def inform_samples(self, sample_ids: list[int], forward_output: torch.Tensor, ta
     @abstractmethod
     def select_points(self) -> tuple[list[int], torch.Tensor]:
         raise NotImplementedError
+
+    def inform_end_of_current_label(self) -> None:
```

Comment:
abstractmethod or not? you have a if that raises not implemented, not sure what this means.

Reply:
Weird inheritance. Now should be clearer